src/celengine/selection.cpp

```diff
@@ -142,7 +142,7 @@ string Selection::getName(bool i18n) const
                     if (parentStar != nullptr)
                     {
                         string buf;
-                        buf = fmt::sprintf("#%d", parentStar->getMainIndexNumber());
+                        buf = fmt::sprintf("#%d", parentStar->getIndex());
```

Comment:
`buf = fmt::sprintf("#%d/%s", parentStar->getIndex(), name);` and remove the line below this one